core/src/main/java/io/varhttp/VarResponseStream.java

```diff
@@ -52,18 +47,20 @@ public OutputStream getOutputStream(String contentType, Charset charset) {
 	@Override
 	public void write(Object object) {
```

Comment:
What do we need, plan to use the other write method for? If we need it we must do validation, otherwise remove it